docs/howtos/ckeditor.md

```diff
@@ -2,7 +2,125 @@
 
-You've [seen how to add a rich text widget to a page and configure styles and toolbar controls](/core-concepts/pages-and-navigation/widgets-singletons-and-areas.md), but sometimes you'll want to go beyond that and configure CKEditor in ways we didn't anticipate. Here's how you can do that.
+You've [seen how to add a rich text widget to a page and configure styles and toolbar controls](/core-concepts/pages-and-navigation/widgets-singletons-and-areas.md), but sometimes you'll want to go beyond that and configure CKEditor for your project's specific needs. Here's how you can do that.
+
+## Changing the allowed HTML tags in rich text
+
+Some CKEditor configurations introduce new HTML tags that might be present in the markup. It looks great in the editor, but when you refresh the page they are gone. What happened?
+
+Apostrophe automatically filters all rich text through the [sanitize-html](https://npmjs.org/package/sanitize-html) module. This prevents XSS attacks and also prevents the page design from being wrecked by bad or just plain ugly markup pasted from desktop applications.
+
+However, sometimes the default configuration isn't working for you, for instance because you want to add `sub` and `sup` to the list of allowed tags.
+
+It's easy to change: just set the `sanitizeHtml` option in `lib/modules/apostrophe-rich-text-widgets/index.js` in your project. Just keep in mind that **if you configure one of the sanitizeHtml options at all, you must configure that option completely.** A common mistake is forgetting to allow `a` elements or the `http` protocol, resulting in very pretty text with no links allowed!
+
+Here are a few common configurations. For more, [see the `sanitize-html` documentation](https://npmjs.org/package/sanitize-html).
+
+### Allow additional HTML tags
+
+The **default** `allowedTags` configuration from `sanitize-html` is:
+
+```javascript
+  allowedTags: [
+    'h3', 'h4', 'h5', 'h6', 'blockquote',
+    'p', 'a', 'ul', 'ol', 'nl', 'li',
+    'b', 'i', 'strong', 'em', 'strike', 'abbr',
+    'code', 'hr', 'br', 'div', 'caption',
+    'table', 'thead', 'tbody', 'tr', 'th', 'td', 'pre', 'iframe'
+  ],
+```
+
+To add tags to that list, you would include all of those tags you want to keep, then add the new ones.  Here is an example adding `sup` and `sub`:
+
+```javascript
+// lib/modules/apostrophe-rich-text-widgets/index.js
+
+module.exports = {
+  // The standard list copied from the module, plus sup and sub
+  sanitizeHtml: {
+    allowedTags: [
+      'h3', 'h4', 'h5', 'h6', 'blockquote',
+      'p', 'a', 'ul', 'ol', 'nl', 'li',
+      'b', 'i', 'strong', 'em', 'strike', 'abbr',
+      'code', 'hr', 'br', 'div', 'caption',
+      'table', 'thead', 'tbody', 'tr', 'th', 'td', 'pre', 'iframe',
+      'sup', 'sub' // ⬅ The new tags
+    ],
+    // ...,
+  }
+};
+```
+
+### Add text styles using classes
```

Comment:
classes should be the first example imo

Reply:
You say this later, coming from the other direction, but here it might be nice to warn the reader that there are _no_ defaults for this set/example